lib/question.js

```diff
@@ -340,6 +377,14 @@ module.exports = {
                     if (ERR(err, callback)) return;
                     courseIssues = ret_courseIssues;
                     data = ret_data;
+
+                    if (submission.workspace) {
+                        logger.info(`data: ${JSON.stringify(data)}`);
+                        logger.info(`... adding _required_file_names`);
+                        data['params']['_required_file_names'] = requiredFileNames;
```

Comment:
Rather than overwriting, I think we normally try and append in case there were other elements that already used this param: https://github.com/PrairieLearn/PrairieLearn/blob/master/elements/pl-file-editor/pl-file-editor.py#L37